src/rdkafka_conf.c

```diff
@@ -1329,9 +1329,11 @@ rd_kafka_anyconf_set_prop (int scope, void *conf,
 				continue;
 
 			/* No match */
+			v = rd_strndup(s, (size_t)(t-s));
 			rd_snprintf(errstr, errstr_size,
-				 "Invalid value for "
-				 "configuration property \"%s\"", prop->name);
+				 "Invalid value \"%s\" for "
```

Comment:
instead of the heap strdup you could either use rd_strndupa() (stack-allocated) or simply:
`"Invalid value \"%.*s\" for ..", (int)(t-s), s, ..`

Reply:
Thanks, learned something new here :)

I updated/force-pushed the change with that, and a test for this situation (based on my rough understanding how the tests are done)